spec/requests/events_spec.rb

```diff
@@ -140,9 +152,15 @@
 
     include_examples "redirect to sign in"
 
-    context "when user is logged in" do
+    context "with logged in user" do
       let(:user) { create :user }
 
+      include_examples "unauthorized access"
+    end
+
+    context "when User is admin" do
```

Comment:
In `profiles_spec`, you use "with admin" but here you write "when User is admin". Is it worth making the context descriptions consistent?